mqtt.md

```diff
@@ -152,8 +154,34 @@ This is unnecessary for the X1 series since it already transmits the full object
 }
 ```
 
+- New Request
+
+```json
+{
+    "user_id": "0",
+    "sequence_id": "0",
+    "pushing": {
+        "version": 1,
+        "command": "pushall"
+    }
+}
+```
+
+```json
+{
+    "user_id": "0",
+    "pushing": {
+        "sequence_id": "0",
+        "version": 1,
+        "push_target": 1
+    }
+}
+```
+
```

Comment:
Isn't this just the same request that I already have which you labelled as "Old request"?

Reply:
Pushall also works with the old request command. I added it because I was wondering if there was anything else. However, the response command is different from the previous pushall response command.

Reply:
But your request looks the same, except you put `sequence_id` at the root?

The slicer does not send commands like this: https://github.com/bambulab/BambuStudio/blob/f6930bb6ab5005184dab3c0fb060394d2fb2f3dc/src/slic3r/GUI/DeviceManager.cpp#L1530-L1533
The slicer clearly always sends `sequence_id` inside `pushing`? Where did you get that request from?

Reply:
I have secured a request parimeter by decompiling the Bambu Handy app. Bambu Studio would like to inform you that certain commands cannot be verified and that the New Request I modified is based on the latest firmware. If the printer version is old, the "New Request" command should not be recognized and the "Old Request" request should be sent out. Similarly, with the latest firmware, the "Old Request" is not recognized.

Reply:
The '// true/false' comment in the 'upgrade_kit' instruction has been deleted from the 'push_all' description, but I don't know how it's used only in the Handy app.

https://github.com/lunDreame/OpenBambuAPI/commit/3234784eeb8acdf79e903347425ef246f3b95c38